server/aesdsocket.c

```diff
@@ -49,14 +67,16 @@ void cleanup_resources() {
         close(custom_socket_fd);
         custom_socket_fd = -1;
     }
-    remove(CUSTOM_LOG_FILE);
+    if (remove(CUSTOM_LOG_FILE) != 0) {
```

Comment:
@Suhas-Reddy-S you shouldn't need to do this in the signal handler anymore, you could do it when you respond to `sig_exit`